<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="Account [2]" uuid="98db0868-4305-4308-8a7a-0af5fa85f4a2">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:D:\WorkSpaces\Program WorkSpace\Python\Project\Python_DataBase_JE\Test_Source\Account.sqlite</jdbc-url>
      <libraries>
        <library>
          <url>file://$APPLICATION_CONFIG_DIR$/jdbc-drivers/Xerial SQLiteJDBC/3.31.1/sqlite-jdbc-3.31.1.jar</url>
        </library>
      </libraries>
    </data-source>
  </component>
</project>